### PR TITLE
 Fix a bug in alias.py

### PR DESCRIPTION
**Pull request by djr7C4**

### CODE REVIEW

File: client/aenea/alias.py [hunk 1]
```diff
@@ -154,7 +155,7 @@ def split(self, text):
         self._update_regex()
         k = 0 # The end of the previous match.
         open_angle_brackets = 0
-
+
```
Comment (nirvdrum) [inline client/aenea/alias.py]: This surely seems petty, but could you remove the unnecessary whitespace addition here please?

Comment (calmofthestorm) [inline client/aenea/alias.py]: It's fine:-)

**Comment by calmofthestorm** [comment]
Thanks!

File: client/aenea/alias.py (hunk 1 above)
Comment (djr7C4) [inline client/aenea/alias.py]: @nirvdrum My editor sometimes inserts extra whitespace.  It seems like a non-issue to me.

Comment (nirvdrum) [inline client/aenea/alias.py]: It seems like Alex doesn't mind, so I won't bring it up again. My editor shows whitespace, so spurious whitespace looks noisy. But, in any event, if you didn't know, you can have git diff show [colored output](http://jblevins.org/log/git-colors) so whitespace is highlighted. And you can use `git add -p` or `git checkout -p` to selectively add or remove parts of a diff so you can avoid adding stuff you didn't intend. If you already knew that, no worries -- I'm not trying to patronizing, I've just found a lot of people don't know and the git defaults aren't the best.

**Comment by calmofthestorm** [comment]
All other things being equal I'd prefer to keep the code clean. If I noticed spurious whitespace in a PR I'd probably ask for cleanup before merge. Obviously I missed this one:-)

**Comment by djr7C4** [comment]
Ok, my editor is configured to auto-delete trailing whitespace on the lines I edit now :)
